docs/components/query-frontend.md

```diff
@@ -85,6 +85,11 @@ config:
   expiration: 0s
 ```
 
+`expiration` Expiration specifies memcached cache valid time , If set to 0s, so using a default of 24 hours expiration time
```

Comment:
Some style issues. Would be good to change to:

`expiration` specifies Memcached cache valid time. If set to 0s, it will use 24 hours expiration time by default.

Reply:
done it.